src/components/hassio-card-content.html

```diff
@@ -0,0 +1,76 @@
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../../bower_components/iron-icon/iron-icon.html'>
+
+<link rel='import' href='./ha-relative-time.html'>
+<link rel='import' href='../util/hass-util.html'>
+
+<dom-module id='hassio-card-content'>
+  <template>
+    <style>
+      iron-icon {
+        color: var(--secondary-text-color);
+        margin-right: 16px;
+        margin-top: 16px;
+        float: left;
+      }
+      iron-icon.update {
+        color: var(--paper-orange-400);
+      }
+      iron-icon.running,
+      iron-icon.installed {
+        color: var(--paper-green-400);
+      }
+      iron-icon.primary {
```

Comment:
The name primary here seems weird. All other class names are describing what it is for (update/running), this one describes what it does (primary color)